clients/apple/lockbook.xcodeproj/xcshareddata/xcschemes/Lockbook-iOS.xcscheme

```diff
@@ -52,13 +52,13 @@
       </BuildableProductRunnable>
       <EnvironmentVariables>
          <EnvironmentVariable
-            key = "RUST_BACKTRACE"
+            key = "LOCKBOOK_DEBUG"
             value = "1"
             isEnabled = "YES">
          </EnvironmentVariable>
          <EnvironmentVariable
-            key = "RUST_LOG"
-            value = "lockbook_core=debug"
+            key = "LOCKBOOK_NO_COLOR"
```

Comment:
Also you can see usage of env var here @Parth

Reply:
Ah right xcode got it